ADALiOS/ADALiOS/ADAuthenticationRequest+WebRequest.m

```diff
@@ -48,8 +50,17 @@ - (void)executeRequest:(NSString *)authorizationServer
      {
          //Prefill the known elements in the item. These can be overridden by the response:
          ADTokenCacheStoreItem* item = [ADTokenCacheStoreItem new];
-         item.resource = _resource;
+         ADAuthenticationSettings* data = [ADAuthenticationSettings sharedInstance];
          item.clientId = _clientId;
+
+         if (data.policy){
+             
+             item.policy = data.policy;
+             
+             
+         }
```

Comment:
This was not my code. I made a CR comment to remove it, I guess it didn't get removed. I have no clue why that would be seen as a valid thing to do.